operations/agent-flow-mixin/alerts/clustering.libsonnet

```diff
@@ -25,5 +25,50 @@ alert.newGroup(
       'Cluster nodes have entered a split brain state.',
       '5m',
     ),
+
+    // Standard Deviation of Lamport clock time between nodes is too high
+    alert.newRule(
+      'ClusterLamportClockDrift',
+      'stddev by (cluster, namespace) (cluster_node_lamport_time) != 0',
+      "Cluster nodes' lamport clocks are not converging.",
+      '5m'
+    ),
+
+    // Nodes health score is not zero.
+    alert.newRule(
+      'ClusterNodeUnhealthy',
+      |||
+        sum by (cluster, namespace, instance) (cluster_node_gossip_health_score) > 0
+      |||,
+      'Cluster node is reporting a health score > 0.',
+      '5m',
+    ),
+
+    // Lamport clock of a node is not progressing at all.
+    alert.newRule(
+      'ClusterLamportClockStuck',
+      'sum by (cluster, namespace, instance) (rate(cluster_node_lamport_time[2m])) == 0',
```

Comment:
nit: use `increase` instead of `rate` here, we don't care what the per-second rate is, we just want to know if it changed in the last two minutes.